conf/database.properties

```diff
@@ -17,11 +17,6 @@ scalar.db.password=cassandra
 # Isolation level used for Consensus Commit. Either `SNAPSHOT` or `SERIALIZABLE` can be specified. The default is `SNAPSHOT`.
 #scalar.db.consensus_commit.isolation_level=
 
-# Serializable strategy used for Consensus Commit.
-# Either `EXTRA_READ` or `EXTRA_WRITE` can be specified. The default is `EXTRA_READ`.
-# If `SNAPSHOT` is specified in the property `scalar.db.consensus_commit.isolation_level`, this is ignored.
-#scalar.db.consensus_commit.serializable_strategy=
```

Comment:
I'm wondering if this property and related implementation should be kept, while EXTRA_WRITE will be rejected, considering the possibility that we might introduce another serializable strategy (e.g., timestamp ordering) in the future. What do you think?

Reply:
Good point, I had the same thought. And I eventually concluded that we can reintroduce the same property and the implementation when we add a new serializable strategy. What do you think?

Reply:
As far as I know, I think it's unlikely that a new strategy will be added anytime soon. So, I agree with you.